### PR TITLE
Rename unithelper to unit_helper for consistency

### LINKED ISSUE #535: Rename unithelper ...
to unit_helper.

### PR DESCRIPTION
**Pull request by lwasser**
closes #535 

## Description

Just a small change to ensure naming consistency in our API. :)

## Type of change

Select the statement best describes this pull request.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This is a documentation update
- [ ] This is a infrastructure update (docs, ci, etc)
- [x] Other (please describe) consistency update!

## Does your PR include tests

If you are fixing a bug or adding a feature, we appreciate (but do not require)
tests to support whatever fix of feature you're implementing.

- [ ] Yes
- [ ] No, i'd like some help with tests
- [x] This change doesn't require tests

## Did you include your contribution to the change log?

- [x] Yes, `changelog.md` is up-to-date.

### CODE REVIEW

**Comment by lwasser** [comment]
NOTE: docs are failing here. that is ok. i'll clean that up in #531

File: src/stravalib/model.py [hunk 1]
```diff
@@ -344,14 +344,14 @@ def validate(cls, value: str) -> Timezone:
...
+
     Examples
     ----------
-    Once you have distance in meters, you can then use the unit_helper module to 
-    convert to other units such as feet 
-    
+    Once you have distance in meters, you can then use the unit_helper module to
+    convert to other units such as feet
+
     >>> from stravalib import unithelper
```
Comment (jsamoocha) [inline src/stravalib/model.py:355]:
```suggestion
    >>> from stravalib import unit_helper
```

File: src/stravalib/model.py [hunk 2]
```diff
@@ -361,6 +361,7 @@ class Distance(_Quantity):
     >>> unithelper.miles(activity.distance)
```
Comment (jsamoocha) [inline src/stravalib/model.py:361]:
```suggestion
    >>> unit_helper.miles(activity.distance)
```

Comment (jsamoocha) [reply src/stravalib/model.py:361]: Also, check line 359 (I was unable to fix it through github)

**Review by jsamoocha — APPROVED** [review]
Thanks!

A few docstring examples still referred to `unithelper`. Maybe a full directory search and replace will ensure we don't miss any others.

Besides that, this can be merged.

**Comment by lwasser** [comment]
thank you @jsamoocha ! i'm glad you reviewed. i did do a global search which is what is so weird.  i should have done a visual check on the examples (which i definitely forgot about!). Merging this

**Pull request merged**